pandas/tests/extension/arrow/arrays.py

```diff
@@ -142,3 +151,29 @@ def any(self, axis=0, out=None):
 
     def all(self, axis=0, out=None):
         return self._data.to_pandas().all()
+
+
+class ArrowBoolArray(ArrowExtensionArray):
+    def __init__(self, values):
+        if not isinstance(values, pa.ChunkedArray):
+            raise ValueError
+
+        assert values.type == pa.bool_()
+        self._data = values
+        self._dtype = ArrowBoolDtype()
+
+    def __repr__(self):
+        return "ArrowBoolArray({})".format(repr(self._data))
```

Comment:
might as well share this method by using `"{cls}({data})".format(cls=type(self).__name__, data=repr(self._data))`